Assignments/Splitwise/CPP/main.cpp

```diff
@@ -0,0 +1,264 @@
+#include <bits/stdc++.h>
+using namespace std;
+class Users
+{
+public:
+    map<string, map<string, float>> users;
+    void addUsers(vector<string>);
+    void getUser(string);
+    void getUser();
+    void addExpense(string, string, string, vector<string>);
```

Comment:
This code is not extensible. Let me pose this question. 
Let us say I wanted to add another expense of a different type. How will you achieve this ?